static/js/util/util.js

```diff
@@ -517,6 +518,20 @@ export function getDisplayName(WrappedComponent: ReactClass<*>) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
+export const wrapWithProps = (addedProps: Object, WrappedComponent: ReactClass<*>) => {
```

Comment:
Nitpick: can you add Flow type information that this function returns a `ReactClass`?